doc/source/working_with_kiwi/legacy_kiwi.rst

```diff
@@ -39,25 +39,25 @@ In order to address all of these the questions came up:
   `How to change/drop features without making anybody unhappy?`
 
 As there is no good way to achieve this in the former code base the
-decision was made to start a rewrite of KIWI with a maintained and
+decision was made to start a rewrite of {kiwi} with a maintained and
 stable version in the background.
 
 Users will be able to use both versions in parallel. In addition, the
-next generation KIWI will be fully compatible with the current format of
+next generation {kiwi} will be fully compatible with the current format of
 the appliance description. This means, users can build an appliance from
 the same appliance description with the legacy and the next generation
-KIWI, if the distribution and all configured features are supported by
-the used KIWI version.
+{kiwi}, if the distribution and all configured features are supported by
+the used {kiwi} version.
 
-This provides an opportunity for users to test the next generation KIWI
+This provides an opportunity for users to test the next generation {kiwi}
 with their appliance descriptions without risk. If it builds and works
-as expected, I recommend to switch to the next generation KIWI. If not,
+as expected, I recommend to switch to the next generation {kiwi}. If not,
 please open an issue on https://github.com/OSInside/kiwi.
 
-The legacy KIWI version will be further developed in maintenance mode.
+The legacy {kiwi} version will be further developed in maintenance mode.
 There won't be any new features added in that code base though.
 Packages will be available at the known place:
-`Legacy KIWI packages <http://download.opensuse.org/repositories/Virtualization:/Appliances>`__
+`Legacy {kiwi} packages <http://download.opensuse.org/repositories/Virtualization:/Appliances>`__
```

Comment:
Same as above, review the hole page, there are many `legacy {kiwi}` or `next generation {kiwi}` in this one. Probably we should rephrase some of those.